resources/views/admin/users/create-or-update.blade.php

```diff
@@ -0,0 +1,72 @@
+@extends('admin.main')
+
```

Comment:
`create-or-update.blade.php` is too long and hard to search.
`user.blade.php` would be cool.